src/FluentHandler.php

```diff
@@ -101,6 +101,32 @@ protected function processFormat(array $record, string $tag): string
         return $tag;
     }
 
+    /**
+     * returns the context
+     * @return array
```

Comment:
typehint array should be added on $context.
return type `array|string` ?

Reply:
done